entry_types/scrolled/package/src/contentElements/inlineVideo/stories.js

```diff
@@ -1,10 +1,10 @@
 import '../frontend';
-import {storiesOfContentElement} from 'pageflow-scrolled/spec/support/stories';
+import {storiesOfContentElement, filePermaId} from 'pageflow-scrolled/spec/support/stories';
 
 storiesOfContentElement(module, {
   typeName: 'inlineVideo',
   baseConfiguration: {
-    id: 'videoGarzweilerDrohne',
+    id: filePermaId('videoFiles', 'interview_toni'),
```

Comment:
This does not work since the seed entry [created for storybook](https://github.com/codevise/pageflow/blob/84c6f7904065fbd5cc46036440a42aab542284f6/entry_types/scrolled/lib/tasks/pageflow_scrolled_tasks.rake#L39) does not include the video file. If it did, we would still have no way to transcode the video in CI, since the seed code now implicitly relies on development mode Resque workers which are not present in CI. Maybe it's ok to render inline videos with unset video files for now?